examples/quickstart/CMakeLists.txt

```diff
@@ -47,14 +57,21 @@ IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     )
   endfunction(add_quickstart_test)
 
+
+  # The problem with the tests below is that it will create failures if someone is building
+  # on an old system with an old default C++.
   if (SIMDJSON_EXCEPTIONS)
-    add_quickstart_test(quickstart quickstart.cpp)
+    # It is not reasonable to test compilation without C++11 since some systems do not have a default
+    # compiler that defaults on C++11 or better. It is not the case of the most recent macOS (Catalina)
+    # in April 2020.
+    # add_quickstart_test(quickstart quickstart.cpp)
```

Comment:
I actually sort of wanted this in here to trigger that exact thing :) "What happens when you don't pass any flags" is an important user experience to validate. On systems that don't pass this the right thing will be to use -E "^quickstart$", I think.

Reply:
For now everything appears to work on mac. So ok.